reVX/least_cost_xmission/cost_creator.py

```diff
@@ -0,0 +1,304 @@
+        self._iso_lookup = iso_lookup
+        self._default_mults = default_mults
+        self._cell_size = CELL_SIZE
+
+    @classmethod
+    def build_cost_rasters(cls, iso_regions_f,
+                           h5_f='costs.h5',
+                           land_use_classes=NLCD_LAND_USE_CLASSES,
+                           iso_mults_f=None, base_line_costs_f=None,
+                           iso_lookup_f=None, power_classes_f=None,
+                           default_mults={}, save_to_h5=True):
+        """
+        Build cost rasters using base line costs and multipliers. Save to
+        h5 file or tiff
+
+        Parameters
+        ----------
+        iso_regions_f : str
+            File with raster of ISO regions
+        h5_f : str
+            H5 file to save costs to
+        land_use_classes : dict
+            NCLD land use codes corresponding to use classes for multipliers
+        iso_mults_f : None | str
+            ISO multipliers JSON file. Use default if None
+        base_line_costs_f : None | str
+            Base line cost per mile for ISOs file. Use default if None
+        iso_lookup_f : None | str
+            JSON file linking ISO region code to name. Use default if None
+        power_classes_f : None | str
+            JSON of line capacities to use for reV power classes. Use default
+            if None
+        default_mults : dict
+            Multipliers for regions not specified in iso_mults_f.
+        save_to_h5 : bool
+            Save output to h5 if true, save to geotiff if false
+        """
+        xc = XmissionConfig(iso_mults_f=iso_mults_f,
+                            base_line_costs_f=base_line_costs_f,
+                            iso_lookup_f=iso_lookup_f,
+                            power_classes_f=power_classes_f)
+
+        xcc = cls(iso_regions_f, land_use_classes, xc.iso_mults,
+                  xc.base_line_costs, xc.iso_lookup, default_mults)
+        mults_arr = xcc.create_multipliers()
+
+        if save_to_h5:
+            xcc._init_h5(h5_f, iso_regions_f)
+        else:
+            xcc._write_geotiff('out/multipliers.tif', xcc._profile, mults_arr)
+
+        for power_class, capacity in xc.power_classes.items():
+            logger.info(f'Calculating costs for class {power_class} using a '
+                        f'{capacity}MW line')
+            blc_arr = xcc.create_base_line_costs(capacity)
+
+            if not save_to_h5:
+                xcc._write_geotiff(f'out/blc_{capacity}MW.tif',
+                                   xcc._profile, blc_arr)
+
+            costs_arr = blc_arr * mults_arr
+            if save_to_h5:
+                xcc._write_layer(h5_f, f'tie_line_costs_{capacity}MW',
+                                 xcc._profile, costs_arr)
+            else:
+                xcc._write_geotiff(f'out/tif_line_costs{capacity}MW.tif',
+                                   xcc._profile, costs_arr)
+
+    def create_multipliers(self):
```

Comment:
silly, but how about `compute_multipliers` instead of create?
Also I think we should add two kwargs here:
`save_layer`
`save_geotiff`
and add the functionality to do the above into this method, then in the class method we would turn save_layer on and pass through the save_geotiff kwarg

Reply:
done